core/domain/feedback_services.py

```diff
@@ -302,11 +302,13 @@ def get_message(thread_id, message_id):
         feedback_models.GeneralFeedbackMessageModel.get(thread_id, message_id))
 
 
+# TODO(Anubhav): Remove this function if the message field in all
```

Comment:
`TODO(anubhavsinha98):`

if --> once

Also I originally asked to file an issue, please do so otherwise we'll forget to do this.